horace_core/admin/hpc.m

```diff
@@ -11,10 +11,11 @@
 % it should be ok to run in on isiscompute.
 %
 % Usage:
-%>>hpc on    -- enable hpc computing extensions
-%>>hpc off   -- disable hpc computing extensions
-%>>hpc       -- print current hpc computing extensions value and the values,
-%               recommended by this function (very crude estimate)
+%    >> hpc on    -- enable hpc computing extensions
+%    >> hpc off   -- disable hpc computing extensions
+%    >> hpc reset -- Set hpc values to the value recommended by this function
+%    >> hpc       -- print current hpc computing extensions value and the values,
+%                    recommended by this function (very crude estimate)
```

Comment:
This does not comprehensively document all the options in the switch statement; no numeric values, no alternatives to reset.